tests/test_market.py

```diff
@@ -271,8 +280,10 @@ def test_market_trade_bid_not_found(market: TwoSidedPayAsBid):
 
 
 @pytest.mark.parametrize("market, offer, accept_offer", [
-    (OneSidedMarket(area=FakeArea("FakeArea")), "offer", "accept_offer"),
-    (BalancingMarket(area=FakeArea("FakeArea")), "balancing_offer", "accept_offer")
+    (OneSidedMarket(area=FakeArea("FakeArea", transfer_fee_pct=0)),
```

Comment:
How about moving setting the transfer_fee_pct=0 directly on the FakeArea class? That way you would not have to make so many changes all over this file.

Reply:
I guess its would be better to keep it like this. In that way I can also test if grid fee is taken into account while immediately placing offers/bids. Like here: https://github.com/gridsingularity/d3a/pull/435/files#diff-9ad3b9e554e26bce69c8a134921b16f2R98